protocol-designer/src/components/modals/FilePipettesModal/index.tsx

```diff
@@ -159,6 +170,38 @@ const makeUpdatePipettes = (
       }
     }
   })
+  const newTiprackUris = new Set(
+    newPipetteArray.flatMap(pipette => pipette.tiprackDefURI)
+  )
+  const previousTiprackLabwares = Object.values(labwareEntities).filter(
+    labware => labware.def.parameters.isTiprack
+  )
+
+  const previousTiprackUris = new Set(
+    previousTiprackLabwares.map(labware => labware.labwareDefURI)
+  )
+
+  // Find tipracks to delete (old tipracks not in new pipettes)
+  previousTiprackLabwares
+    .filter(labware => !newTiprackUris.has(labware.labwareDefURI))
+    .forEach(labware => dispatch(deleteContainer({ labwareId: labware.id })))
+
+  // Create new tipracks that are not in previous tiprackURIs
+  newTiprackUris.forEach(tiprackDefUri => {
+    if (!previousTiprackUris.has(tiprackDefUri)) {
+      const adapterUnderLabwareDefURI = newPipetteArray.some(
+        pip => pip.name === 'p1000_96'
```

Comment:
let's avoid using `pip` 😁 

```suggestion
        pipette => pipette.name === 'p1000_96'
```

Reply:
i got lazy 😆 😅